lineman/app/components/group_page/cover_photo_form/cover_photo_form.haml

```diff
@@ -0,0 +1,21 @@
+.group-cover-photo-modal
+  .lmo-disabled-form{ng-show: 'isDisabled'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
```

Comment:
Hm, yeah, and I copied this from some other place in the code, which means there's other places it's not right. There's that BaseFormModal class cropping up again...